omero/sysadmins/version-requirements.txt

```diff
@@ -0,0 +1,1883 @@
+
+.. list-table::
+    :header-rows: 1
+
+    * - Version
+      - Release date
+      - Upstream support
+      - OME (build)
+      - OME (deploy)
+      - Details
+    * - 5.0
+      - from Feb 2009
+      - to Feb 2012
+      - Dropped
+      - Dropped
+      - `Reference <https://www.debian.org/releases/lenny/>`__
+    * - 6.0
+      - from Feb 2011
+      - to Feb 2016
+      - Supported
+      - Supported
+      - `Reference <https://www.debian.org/releases/squeeze/>`__
+    * - 7.0
+      - from May 2013
+      - TBA
+      - Recommended
+      - Recommended
+      - `Reference <https://www.debian.org/releases/wheezy/>`__
+    * - 8.0
+      - TBA
+      - TBA
+      - Upcoming
+      - Upcoming
+      - `Reference <https://www.debian.org/releases/jessie/>`__
+
+Stable releases 7.x and 6.x are supported. There is no CI testing for
+any version, and some developer use of 7.x.
+
+Linux (Debian testing and unstable)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Due to the fast pace of Debian developement, it only really makes
+sense to support the last one or two stable releases; the testing and
+unstable requirements will be covered in the dependencies, below.
+
+Hardware support
+----------------
+
+
+Microsoft Windows
+^^^^^^^^^^^^^^^^^
+
+Supports amd64 (64-bit) and i386 / x86 (32-bit) for all supported
+versions; most systems are today running x64 with most hardware being
+x64, but there is still a niche of x86 users, estimated <25% and
+falling globally (OMERO usage stats here?)
+
+Unix (MacOS X)
+^^^^^^^^^^^^^^
+
+Currently all supported versions are amd64 (64-bit) only.
+
+Unix (FreeBSD)
+^^^^^^^^^^^^^^
+
+Supports amd64 (64-bit) and i386 / x86 (32-bit) for all supported
+releases. Other platforms are supported, but may require manual
+building of prerequisites.
+
+Linux (RHEL and CentOS)
+^^^^^^^^^^^^^^^^^^^^^^^
+
+Version 6 supports amd64 (64-bit) and i386 / x86 (32-bit). Version 7
+supports amd64 (64-bit) only.
+
+Linux (other)
+^^^^^^^^^^^^^
+
+Most distributions support i386 / x86 (32-bit) as well, while some
+support additional architectures such as 32-bit and/or 64-bit arm and
+powerpc.
+
+Additional notes
+^^^^^^^^^^^^^^^^
+
+The OMERO server is 64-bit-only due to the large memory requirements.
+On the client side, there is a requirement for 32-bit support for
+Windows clients and to a lesser extent on Linux, and zero on MacOS X
+at present. Dropping 32-bit support for OMERO.server and C++ Ice and
+other builds should be considered, given the vast reduction in the
+support burden and steadily dropping need for these builds.
+
+Dependencies
+------------
+
+The following subsections detail the versions of each dependency
+needed by OMERO which are supported by both its upstream developers
+(for security and general updates) and by OME for OMERO building and
+server and client deployment.
+
+.. note::
+    Versions in brackets are in development distributions and may
+    change without notice.
+
+Package lists
```

Comment:
This is primarily for use in updating the information below and for adding additional packages; it's a link to the package lists for each system we're potentially interested it.  It's not really directly useful for sysadmins.  It could be turned into sphinx comments to keep it out of the doc, but it might possibly be of use to others.  Certainly happy to change it if desired.